stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/attachments/StreamAttachmentFactories.kt

```diff
@@ -34,18 +35,32 @@ public object StreamAttachmentFactories {
      */
     private const val DEFAULT_LINK_DESCRIPTION_MAX_LINES = 5
 
+    /**
+     * Instructs the Giphy to fill all of the available space.
+     */
+    public const val GIPHY_FILL_MAX_SPACE: Float = -1f
+
     /**
      * Default attachment factories we provide, which can transform image, file and link attachments.
      *
      * @param linkDescriptionMaxLines - The limit of how long the link attachment descriptions can be.
+     * @param giphyInfoType Used to modify the quality of the rendered Giphy attachments.
+     * @param giphyUpscaleFactor The amount the Giphy will upscaled. By default this is set to 1,
+     * meaning that the each Giphy pixel will take exactly 1 screen pixel.
+     *
      * @return A [List] of various [AttachmentFactory] instances that provide different attachments support.
      */
     public fun defaultFactories(
         linkDescriptionMaxLines: Int = DEFAULT_LINK_DESCRIPTION_MAX_LINES,
+        giphyInfoType: GiphyInfoType = GiphyInfoType.ORIGINAL,
```

Comment:
The factories are contained within `ChatTheme`, so I'm not sure if we can leverage the VMs to control them at runtime easily.

As a compromise, I've instilled these parameters here so that users can still adjust it to their liking.

Reply:
These things shouldn't really change in runtime much, so it's fine to keep them here

Reply:
And even if they need to - people can build UI that re-builds these factories based on a remember/state block, so it'll recompose and change